### PR TITLE
Fix indentations

### PR DESCRIPTION
**Pull request by nzwsch**

### CODE REVIEW

File: asset_management/sinatra_assetpack.md [hunk 1]
```diff
@@ -127,26 +127,27 @@ The `serve` method can be specified so that the folder from which the
...
+    # You can also do this: 'js/*.js'
+  ]
+
+  serve '/css', :from => 'assets/stylesheets'
+  css :application, [
+    '/css/jqueryui.css',
+    '/css/reset.css',
+    '/css/foundation.css',
+    '/css/app.css'
+   ]
```
Comment (locks) [inline asset_management/sinatra_assetpack.md]: shouldn't this be one space behind?

Comment (nzwsch) [inline asset_management/sinatra_assetpack.md]: @locks oops, I didn't realize that.

**Comment by locks** [comment]
Can you squash the commits please?

**Comment by nzwsch** [comment]
@locks Can I merge 5903340?

**Pull request merged**